django/thunderstore/core/settings.py

```diff
@@ -429,6 +429,10 @@ def show_debug_toolbar(request):
     ],
 }
 
+# Thumbnails
+
+THUMBNAIL_PRESERVE_EXTENSIONS = ('png',)
```

Comment:
@nihaals My example icon grew from 12 KB (85% jpg) to 51 KB (original png). While it is considerably bigger, it's still reasonably small IMO. That said, just upping the jpg quality from 85% to 95% is probably just as good in practice, but still resulting in smaller file sizes.

```suggestion
THUMBNAIL_QUALITY = 85
```

Reply:
We do show our thumbnails quite large, so at least increasing the quality makes sense. I think even if we're dealing with low resolution images, decreasing file size is still important and a 4.25x increase is quite significant (although file size increases can vary *significantly* depending on the image).

@MythicManiac What do you think?

If there's still not an easy answer, we could compare image quality and file size for `85% JPEG`, `90/95% JPEG` and `PNG` for some of the most popular packages (or a random selection) to get a better idea. I don't have any analytics relating to mobile users or average page load time so it's hard to put into perspective how much of a difference this would make, but with 24 images per page it will have a larger impact than it seems.